guide/sharding/extended.md

```diff
@@ -285,52 +285,37 @@ Now, run this code, and you will surely get a result that looks like the followi
 ]
 ```
 
-While this result isn't *necessarily* bad or incorrect, it's simply a raw object that got `JSON.parse()`'d and `JSON.stringify()`'d over, so all of the circular references are gone. More importantly, The object is no longer a true <branch version="11.x" inline>`Emoji`</branch><branch version="12.x" inline>`GuildEmoji`</branch> object as provided by discord.js. This means none of the convenience methods usually provided to you are available. If this is not a concern to you, you can effectively skip the rest of this section. However, this tutorial should cover it regardless! Let's remedy this issue, shall we?
+While this result isn't *necessarily* bad or incorrect, it's simply a raw object that got `JSON.parse()`'d and `JSON.stringify()`'d over, so all of the circular references are gone. More importantly, The object is no longer a true <branch version="11.x" inline>`Emoji`</branch><branch version="12.x" inline>`GuildEmoji`</branch> object as provided by discord.js. *This means none of the convenience methods usually provided to you are available.* If this is a problem for you, you will want to handle the item *inside* the `broadcastEval`. Conveniently, the `findEmoji` function will be ran inside it, so you should execute your relevant methods there, before the object leaves the context.
 
 <branch version="11.x">
 
-```diff
-function findEmoji(id) {
--   const emoji = this.emojis.get(id);  
-+   const temp = this.emojis.get(id);
--   if (!emoji) return null;
-+   if (!temp) return null;
-+
-+   // Clone the object because it is modified right after, so as to not affect the cache in client.emojis
-+   const emoji = Object.assign({}, temp);
-+   // Circular references can't be returned outside of eval, so change it to the id
-+   if (emoji.guild) emoji.guild = emoji.guild.id;
-+   // A new object will be constructed, so simulate raw data by adding this property back
-+   emoji.require_colons = emoji.requiresColons;
-+
+```js
+function findEmoji(nameOrID) {
+	const emoji = this.emojis.get(nameOrID) || this.emojis.find(e => e.name.toLowerCase() === nameOrID.toLowerCase());
+	if (!emoji) return null;
+	// If you wanted to delete the emoji with Discord.js, this is where you would do it. Otherwise, don't include this code.
```

Comment:
```suggestion
	if (!emoji) return;
	// If you wanted to delete the emoji with discord.js, this is where you would do it. Otherwise, don't include this code.
```